README.rst

```diff
@@ -56,14 +56,27 @@ More information on `our website <http://www.django-cms.org>`_.
 Requirements
 ************
 
-django CMS requires Django 1.8, and Python 2.7, 3.3 or 3.4.
+See the `Python/Django requirements for the current release version
+<http://http://docs.django-cms.org/en/stable/#software-version-requirements-and-release-notes>`_ in our documentation.
+
+See the `installation how-to guide for an overview of some other requirements and dependencies of the current release
+<http://docs.django-cms.org/en/stable/how-to/install.html>`_
 
 *************
 Documentation
 *************
 
-Please head over to our `documentation <http://docs.django-cms.org/>`_ for all
-the details on how to install, extend and use the django CMS.
+We maintain documentation for several versions of the project. Key versions are:
+
+* `stable <http://http://docs.django-cms.org>`_ (default), for the **current release** version
```

Comment:
No - not at present, anyway. If you use https:
<img width="624" alt="screenshot 2017-01-03 12 49 17" src="https://cloud.githubusercontent.com/assets/86222/21608219/1cfbbcae-d1b3-11e6-9a70-fabcc9249622.png">